MainModule/Client/UI/Default/Settings.lua

```diff
@@ -65,6 +65,21 @@ return function(data)
 				toggle.Text = text
 			end
 		};
+		{
+			Text = "Hide Chat Commands: ";
+			Desc = "- Hide your commands run from the chat from others";
```

Comment:
Maybe reword for clarity?
```suggestion
			Desc = "- Hide commands ran from the chat";
```